src/components/Navigation.js

```diff
@@ -65,7 +66,11 @@ const Navigation = () => {
             href="https://docs.google.com/forms/d/e/1FAIpQLSe5M-kEntAmPHnHNcerN6F8G5eevJs6zQJgEt3X3hcj_eNV3w/viewform"
             className="mb-0 py-1 px-3 no-underline ml-auto text-2xl whitespace-nowrap bg-winc-white w-full text-center !text-winc-black !font-regular hover:!text-winc-pink hover:cursor-pointer"
           >
-            join now
+            <Button
```

Comment:
for right now, don't test the button in the nav bar, I will create a separate issue for that. Just test it in another file and delete the test code before pushing